### ISSUE TITLE
Maintenance Release to remove Full Screen button from iPad iOS 12

### ISSUE THREAD

**Issue by samreid** [phetsims/joist#525]
See https://github.com/phetsims/joist/issues/524.  @ariel-phet asked if @jonathanolson can lead this maintenance release, and I volunteered to assist.  @ariel-phet recommended this be completed by the end of the week next week, so I'll elevate the priority.  @jonathanolson said he would like to fine-tune some documentation before we begin, please notify me when ready.

**Comment by jonathanolson** [comment]
Commits above are just patching some non-published release branches as part of a test (and so we don't miss them).

**Comment by samreid** [comment]
I asked @ariel-phet on slack:

> Apple released iOS 12.0.1 yesterday.  Do you want QA to test fullscreen behvaior on that new version before @jonathanolson and I begin maintenance releasing?

**Comment by samreid** [comment]
Sam Reid [12:07 PM]
Apple released iOS 12.0.1 yesterday.  Do you want QA to test fullscreen behvaior on that new version before @jonathanolson and I begin maintenance releasing?

Ariel Paul [12:20 PM]
Probably a good idea

Sam Reid [12:33 PM]
OK we’ll reach out to KW

Sam Reid [12:32 PM]
Hi there, Jonathan and I are getting ready to maintenance release for the iOS 12 fullscreen issue.  But we noticed Apple released iOS 12.0.1, and Ariel was wondering if you can test on that before we proceed.

Do you have any 12.0.1 devices at the moment?

Kathryn Woessner [12:41 PM]
Kathy has all our iPads but iOS 10.  I don't know when she plans to bring it back.

**Comment by samreid** [comment]
Sam Reid [12:49 PM]
KW doesn’t have any iOS 12 devices at the moment, it sounds like Kathy has them.  Would you prefer Jonathan and I reschedule the maintenance release until we’ve had time to check on that, or do we want to take the fullscreen button out of iOS regardless?

Ariel Paul [1:14 PM]
We can probably wait

Sam Reid [1:15 PM]
OK we’ll reschedule, thanks!

**Comment by samreid** [comment]
On hold until we've tested iOS 12.0.1.

**Comment by KatieWoe** [comment]
Conversation with @ariel-phet 
```
Kathryn Woessner [11:18 AM]
iOS 12 full screen is being removed in a maintenance release right?  It is on published sims at the moment.

Ariel Paul [11:22 AM]
Probably....I believe we want to double check with the new update before moving forward with that maintenance release...
Just so we have full data
since it looks like some things were fixed in iOS 12.01

Kathryn Woessner [11:59 AM]
Laura is working on the reverse array issue on 12.01 and she found the full screen problem.

Ariel Paul [1:23 PM]
Ok, let Sam know the full screen issue is still present in 12.01
```
@samreid

**Comment by KatieWoe** [comment]
@jbphet confirmed on States of Matter 1.1.0-rc.3

**Comment by samreid** [comment]
Notes from the maintenance release process:

We need to exclude these sims:
John Travoltage bumped to 1.5.0
States of Matter 1.1

NOTE: delete .DS_STORE or something like that before starting.

Maintenance.checkBranchStatus()
Maintenance.createPatch( 'joist', 'https://github.com/phetsims/joist/issues/524' )
Maintenance.addNeededPatchesBefore( 'joist', '3b6ccf3ff0966983c25a7a1603292d8a261657cb' )

// These were patched manually
Maintenance.removeNeededPatch( 'john-travoltage', '1.5', 'joist' ) 
Maintenance.removeNeededPatch( 'states-of-matter', '1.1', 'joist' ) 

Maintenance.addPatchSHA( 'joist', '3b6ccf3ff0966983c25a7a1603292d8a261657cb' )
Maintenance.applyPatches()

// Something like Maintenance.checkoutBranch( 'area-builder', '1.1' ) but for acid-base-solutions and its branch, then create patch sha
Maintenance.addPatchSHA( 'joist', '764d40522e9a844352840de8d0386d03a54857df' )
Maintenance.applyPatches()

Maintenance.checkoutBranch( 'area-builder', '1.1' )
Maintenance.addPatchSHA('joist','c5ccd9fc6faaa8e18192674fa12143fc566d9448');
Maintenance.applyPatches()

//... 
Maintenance.addPatchSHA('joist','2f8fd4a723e94c5f88682ab712f5a959a1b288d1');
Maintenance.applyPatches()

Maintenance.addPatchSHA('joist','11402fad25eabbba4a615ba5adef2068ccfc79f5');
Maintenance.applyPatches()

// Run this, could take more than an hour
Maintenance.updateDependencies()

// if no errors, then go to next step:, could take several hours.
// BUT FIRST, manually patch balloons and static electricity 1.3-phetio (done)
// BUT ALSO FIRST, check out that branch 1.3-phetio and git pull to make sure my working copy gets the changes (done)
Maintenance.deployReleaseCandidates()

Maintenance.linkList()

Create an issue in the special-ops repo, to keep PhET-iO URLs private, see for example: https://github.com/phetsims/special-ops/issues/114

10-30 seconds, make sure it launches on Win/Chrome and make sure the fullscreen button appears. Then also run on iOS and make sure there is no fullscreen button.

We'll need to do this one manually (done)
        "repo": "balloons-and-static-electricity",
        "branch": "1.3-phetio",

**Comment by samreid** [comment]
We finished this issue in https://github.com/phetsims/special-ops/issues/122, closing.
